src/bin/server.ts

```diff
@@ -129,7 +130,7 @@ interface Args {
 
   driver.on("driver ready", async () => {
     try {
-      server = new ZwavejsServer(driver, { port: wsPort });
+      server = new ZwavejsServer(driver, { port: wsPort, host: args.host });
```

Comment:
let's use the same pattern as we are doing for the port argument at line 29, e.g.:
```
let wsHost = "0.0.0.0";
if (args["host"]) {
  wsHost = args["host"]
}
```
and then pass that to the function. There are also two log statements, one in `src/bin/server` and one in `src/lib/server` that reference the port, I think we should add the host into those statements for completeness

Reply:
Should the `host` be optional? I see that port isn't 🤷

Reply:
yes, port is optional too it's just not typed correctly, so if you want to fix that while you are at it, ❤️

Reply:
Should be done 👍